lib/reversetunnel/srv.go

```diff
@@ -1166,6 +1165,7 @@ func newRemoteSite(srv *server, domainName string, sconn ssh.Conn) (*remoteSite,
 			Clock:     srv.Clock,
 			Client:    remoteSite.remoteAccessPoint,
 		},
+		// TODO: add OpenSSHCA?
```

Comment:
Leftover comment? I also don't think we should add OpenSSH CA here.

Reply:
Yep, thanks will remove